pkg/platform/version.go

```diff
@@ -61,15 +61,19 @@ func LatestCompatibleVersion(ctx context.Context) (string, error) {
 			return semver.Version{}, false
 		}
 
+		// skip the pre-releases (alpha, rc, etc.)
+		if len(ghVersion.Pre) != 0 {
+			return semver.Version{}, false
```

Comment:
Why return here instead of continue ? Won't that stop finding latest release as soon as it finds a pre release.

Reply:
No, actually this is a callback function which runs on every item of the `releases` array provided in the argument of `FilterMap()` method. If the `ghVersion` is not eligible, we return false as a second argument which basically tells the `FilterMap` method to filter it out i.e. reject it. Here is the [reference code](https://github.com/samber/lo/blob/master/slice.go#L57) for FilterMap.